Library/Homebrew/os/mac/linkage_checker.rb

```diff
@@ -77,6 +78,11 @@ def check_undeclared_deps
         a <=> b
       end
     end
+    extraneous_deps = declared_dep_names.reject do |full_name|
+      name = full_name.split("/").last
+      Formula[name].bin.directory? || @brewed_dylibs.keys.map { |x| x.split("/").last }.include?(name)
```

Comment:
`next true if Formula[name].bin.directory?` and put the `map` on a new line so it is a little shorter.

Reply:
perhaps `next true unless ...?`

Reply:
nope, `next true if...` :)